wdl-lint/tests/lints/redundant-input-assignment-wdl-1.1/source.wdl

```diff
@@ -1,5 +1,5 @@
-#@ except: DescriptionMissing, MissingRequirements
-#@ except: RuntimeSectionKeys, MissingOutput, MissingMetas
+#@ except: MetaDescription, RequirementsSection
+#@ except: ExpectedRuntimeKeys, OutputSection, MetaSections
```

Comment:
@a-frantz - Previously these were generally alphabetical, though that wasn't quite the case here. Do you see any reason to reorder all of these to be alphabetical?

Reply:
Having gone through the rest of these, there were more out-of-order than I remembered. There's also the question of formatting, some rules don't have spaces after the commas.

Reply:
> some rules don't have spaces after the commas.

should be pretty easy. we are only targetting `#@ except: ` for alphabetical order too ?

Reply:
I don't see a need for them to be alphabetical or consistent with the space after the comma. It could be nice to have that consistent, but I don't think it's worth anyone's effort to enforce now or to maintain in the future.

Reply:
although 🤔 should `sprocket format` handle this?

Reply:
I'm not going to push for it here. I do think `sprocket format` should take an opinionated approach, at least to spaces after the commas. This would give us a decent test case for that functionality as well.

Reply:
issue created https://github.com/stjude-rust-labs/wdl/issues/410

Reply:
> although 🤔 should `sprocket format` handle this?

ah didn't think about that i just `cdo` over all occurances.

but on the second note, we do have some incorrect indentation/whitespaces necessary for checking the rules and doing format over all those files won't be ideal?

Reply:
> but on the second note, we do have some incorrect indentation/whitespaces necessary for checking the rules and doing format over all those files won't be ideal?

This is true. We might not want to actually run `sprocket format` on the test directories for this reason. But it's a problem for another day, because that functionality doesn't exist yet!